src/python_routing_v02/.ipynb_checkpoints/compute_nhd_routing-checkpoint.py

```diff
@@ -0,0 +1,343 @@
+## Basic imports
```

Comment:
This looks like it was accidentally dragged into the commit...

Reply:
resolved - this checkpoint directory was deleted